guides/common/modules/con_security-features.adoc

```diff
@@ -0,0 +1,43 @@
+= Security features in {Project}
+
+When deploying {Project}, consider implementing security features that enhance the overall security of your system.
+{Project} supports multiple security mechanisms, including FIPS mode, `fapolicyd`, and SELinux, to provide additional layers of protection.
+
+Federal Information Processing Standards (FIPS) Mode::
+{Project} supports running on FIPS-enabled {RHEL} hosts to comply with security standards for cryptographic modules. 
+However, {Project} itself does not currently support running in FIPS mode.
++
+FIPS mode enforces the use of validated cryptographic algorithms and modules.
+Enabling FIPS mode ensures that all cryptographic operations within the system adhere to strict security standards.
++
+For more information, see {RHELDocsBaseURL}9/html/security_hardening/switching-rhel-to-fips-mode_security-hardening[Switching RHEL to FIPS mode] in _{RHEL}{nbsp}9 Security hardening_.
+
+File Access Policy Daemon (`fapolicyd`)::
+By enabling `fapolicyd` on your {Project} Server, you can monitor and control file and directory access, providing an additional security layer.
++
+This feature helps prevent unauthorized code execution and enhances system integrity. You can enable or disable `fapolicyd` on both {Project} and {SmartProxyServer} at any time, depending on your security requirements.
++
+For additional information about `fapolicyd`, see {RHELDocsBaseURL}9/html/security_hardening/assembly_blocking-and-allowing-applications-using-fapolicyd_security-hardening[Blocking and allowing applications by using fapolicyd] in _Security hardening_.
++
+For instructions on enabling `fapolicyd`, see {InstallingServerDocURL}installing_satellite_server_in_a_connected_network_environment/installing_server_connected_{project-context}#using-fapolicyd-on-server_{project-context}[Using fapolicyd on Satellite Server
+] in _{InstallingServerDocTitle}_.
+
+https://docs.redhat.com/en/documentation/red_hat_satellite/6.16/html/installing_satellite_server_in_a_connected_network_environment/installing_server_connected_satellite#using-fapolicyd-on-server_satellite
+
+Security-Enhanced Linux (SELinux)::
+{Project} supports SELinux, a mandatory access control system that restricts system access and reduces the risk of security vulnerabilities.
+SELinux enforces policies that define which processes can access specific system resources, providing protection against unauthorized access and exploitation.
++
+Running {Project} with SELinux in enforcing mode is recommended to maintain a high-security environment.
+It ensures that access control policies are strictly enforced, minimizing the risk of privilege escalation or unauthorized actions.
++
+Using SELinux in enforcing mode is required for {Project} Server to be installed on your environment. For additional information, see {InstallingServerDocURL}preparing_your_environment_for_installation_{project-context}#system-requirements_{project-context} in _{InstallingServerDocTitle}_.
```

Comment:
This contradicts each other: first it's recommended but then it's required. In the installation guide we say:
> SELinux must be enabled, either in enforcing or permissive mode.

In other words, it's currently recommended (and IMHO that's still correct).